spec/presenters/single_content_item/single_content_item_presenter_spec.rb

```diff
@@ -0,0 +1,238 @@
+RSpec.describe SingleContentItemPresenter do
```

Comment:
Do we need the folder `/single_content_item_presenter`?

Reply:
Sorry forgot to remove this mid-edits. Done https://github.com/alphagov/content-data-admin/pull/328/commits/41d4a7f60abb0a8c921c44ec923654fc1e51a164